Gemfile

```diff
@@ -1,11 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf',  '~> 3.2'
-gem 'chefspec',   '~> 4.2'
-gem 'foodcritic', '~> 4.0'
-gem 'rubocop',    '~> 0.30.1'
+gem 'berkshelf', '~> 4.0'
+gem 'chef-zero', '= 4.5'
```

Comment:
If you bump all the other deps to current you'll get the chef-zero 4.7 release which pins rack and solves this problem